case_studies/psf_variation/train_models.sh

```diff
@@ -1,27 +1,20 @@
 #!/bin/bash
 
-# Runs the experiments for the BLISS PSF variation paper.
+# This script runs the experiments from the BLISS spatially-variant PSF paper.
 
-# train base model
-bliss -cp ~/bliss/case_studies/psf_variation -cn config mode=train \
-    train.trainer.logger.name=PSF_MODELS \
-    train.trainer.logger.version=base_model \
-    encoder.image_normalizer.concat_psf_params=False \
-    cached_simulator.cached_data_path=/data/scratch/aakash/train_single_94-1-12
+# Generate new data
+bash ~/bliss/scripts/generate_data_in_parallel.sh -n 32 -cp ~/bliss/case_studies/psf_variation/conf -cn psf_aware
+bash ~/bliss/scripts/generate_data_in_parallel.sh -n 32 -cp ~/bliss/case_studies/psf_variation/conf -cn single_field
 
-# train psf-unaware model
-bliss -cp ~/bliss/case_studies/psf_variation -cn config mode=train \
-    train.trainer.logger.name=PSF_MODELS \
-    train.trainer.logger.version=multi_field_psf_unaware \
-    encoder.image_normalizer.concat_psf_params=False \
-    cached_simulator.cached_data_path=/data/scratch/aakash/train_multi_field
+# Train single-field model
+bliss -cp ~/bliss/case_studies/psf_variation/conf -cn single_field mode=train
 
-# train concat params only model
-bliss -cp ~/bliss/case_studies/psf_variation -cn config mode=train \
-    train.trainer.logger.name=PSF_MODELS \
-    train.trainer.logger.version=multi_field_psf_params_only \
-    encoder.image_normalizer.concat_psf_params=True \
-    cached_simulator.cached_data_path=/data/scratch/aakash/train_multi_field
+# Train psf-unaware model
+bliss -cp ~/bliss/case_studies/psf_variation/conf -cn psf_unaware mode=train
 
-# run the evaluation notebook
-jupyter nbconvert --execute evaluate_models.ipynb --to html
+# Train psf-aware model
+bliss -cp ~/bliss/case_studies/psf_variation/conf -cn psf_aware mode=train
+
+# Run evaluation and generate figures
+python evaluate_models.py --run_eval --plot_eval --run_calibration --plot_calibration --data_path=/data/scratch/aakash/multi_field
+python evaluate_models.py --run_eval --plot_eval --run_calibration --plot_calibration --data_path=/data/scratch/aakash/single_field
```

Comment:
Could you call the ipynb files in `case_studies/psf_variation` too? (Or if we no longer need those notebooks, could we remove them?)

Reply:
Oh, there's actually just one left: `find_param.bins.ipynb`.

Reply:
Oh and I see: this notebooks doesn't do much except find the thresholds that are included elsewhere

Reply:
Yeah it's just finding magnitude quantiles to use for binning in metrics and the calibration plots. I kept it in for reference but it's not really relevant to the main results